content/integrations/sonarcloud-integration.md

```diff
@@ -0,0 +1,52 @@
+---
+title: SonarCloud integration
+description: How to integrate your workflows with SonarCloud using codemagic.yaml
+weight: 2
+---
+**SonarCloud** is a cloud-based code quality and code security service for projects hosted in cloud based Git repositories such as GitHub, Bitbucket, GitLab and Azure DevOps. It can be used as part of your CI/CD workflow to analyse your code each time you commit new code. 
```

Comment:
```suggestion
**SonarCloud** is a cloud-based code quality and code security service for projects hosted in cloud-based Git repositories such as GitHub, Bitbucket, GitLab and Azure DevOps. It can be used as part of your CI/CD workflow to analyse your code each time you commit new code. 
```